news/_posts/2022-12-03-playtest-20221203.md

```diff
@@ -23,40 +23,40 @@ There have been tons of massive changes under the hood, and with reduced contrib
 * Our new netcode brings major improvements to multiplayer. Your games should feel much smoother and units more responsive in both low and high latency settings.
 
 <figure>
-  <video width="800" autoplay loop muted>
+  <video autoplay loop muted>
     <source src="{{ '/images/news/20221203-td.mp4' | relative_url }}" type="video/mp4">
-    <source src="{{ '/images/news/20221203-td.ogv' | relative_url }}" type="video/ogg">
-    <img src="{{ '/images/news/20221203-td.png' | relative_url }}" width="800" />
+    <source src="{{ '/images/news/20221203-td.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20221203-td.png' | relative_url }}" />
```

Comment:
Have you tried running the website? Does it look the same?

Reply:
Yes, it looks the same. Tested on FF, Chrome and a webkit browser on Linux.

Reply:
so it scales to the size of the image?

Reply:
the `video` and `img` tag are limited to 600px in width: 
https://github.com/OpenRA/OpenRAWebsiteV3/blob/3ef268fc85ad3d526a939f4088559b0c54706de1/_sass/includes/developer-update.scss#L28-L31
https://github.com/OpenRA/OpenRAWebsiteV3/blob/3ef268fc85ad3d526a939f4088559b0c54706de1/_sass/includes/developer-update.scss#L1

Reply:
That can't be right. In this newspost we had 700px size, then boosted to 800px and it looked bigger

Reply:
I'm on safari. However abc was the one who did it and he probably used firefox or another of the main 3.

Reply:
I just say let's leave it as is

Reply:
Okay. False alarm the sizes didn't change. Just looked at the diff between the images and both 700px and 800px were the same size. It was just other factors and confirmation bias that misdirected us

Reply:
Yes, the inline styles and attributes don't make a difference because there is a `max-width` declaration in the CSS. I think these should be cleaned up from all of the posts. I'll drop the commit from here and open another PR.